src/components/profile/Profile.vue

```diff
@@ -65,7 +65,7 @@
         </RouterLink>
       </header>
       <ReportingStructure
-        v-if="manager || directs.length > 0"
+        v-if="false && (manager || directs.length > 0)"
```

Comment:
What is this for?

Reply:
That's not there anymore and I need to change this properly.